README.md

```diff
@@ -1,39 +1,61 @@
-| `test_fmf_plan`              | A fmf plan which will be selected from tmt repository.                                                                            | all                           |
-| `tests_tmt_ref`              | A tmt tests branch which will be used for tests                                                                                   | master                        |
 | `compose`                    | Compose to run tests on. [Available composes.](https://api.dev.testing-farm.io/v0.1/composes)                                     | Fedora                        |
-| `create_issue_comment`       | If GitHub action will create a github issue comment.                                                                              | false                         |
-| `pull_request_status_name`   | GitHub pull request status name                                                                                                   | Fedora                        |
 | `arch`                       | Define an architecture for testing environment                                                                                    | x86_64                        |
-| `env_vars`                   | Environment variables for test env, separated by ;                                                                                | empty                         |
-| `env_secrets`                | Environment secrets for test env, separated by ;                                                                                  | empty                         |
+| `variables`                  | Environment variables for test env, separated by ;                                                                                | empty                         |
+| `secrets`                    | Environment secrets for test env, separated by ;                                                                                  | empty                         |
+
+### Test Artifacts
+
 | `copr`                       | Copr name to use for the artifacts                                                                                                | epel-7-x86_64                 |
-| `test_artifacts`             | `fedora-copr-build` artifacts for testing environment, separated by ;                                                             | empty                         |
+| `copr_artifacts`             | `fedora-copr-build` artifacts for testing environment, separated by ;                                                             | empty                         |
+
+
+### Miscellaneous
+
+| `create_issue_comment`       | If GitHub action will create a github issue comment.                                                                              | false                         |
```

Comment:
This is a question to @phracek:

Would `create_issue_comment`  be supported in further versions of testing-farm-as-a-github-action? It seems, that this variable is currently not used. If my assumption is true I would suggest removing this input, for now, so the user is not confused.

Reply:
I totally forgot to add this functionality.

I will do it as a separate pull request.

Reply:
https://github.com/sclorg/testing-farm-as-github-action/pull/23